docs/ota-client-guide/modules/ROOT/pages/change-signature-thresholds.adoc

```diff
@@ -0,0 +1,52 @@
+= Change signature thresholds
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+A threshold cryptosystem is one of the ways in which we protect your software repository. By default, the threshold number is one. This means that you can use any of the valid keys to sign your software versions. For more security, you can also change the threshold number of your Root and Targets keys, as well as delegations.
+
+*To change the thresholds of Root and Targets keys:*
+
+. In the *roles* folder of your local software repository, open the *unsigned* folder, and then open the `root.json` file.
+. Depending on the threshold that you want to change, in the `signed` block, do the following:
+
+** To change the threshold of Root keys:
+.. In the `root` subsection, for the `keyids` object, specify the list of valid keys that can be used for signing.
++
+You can find the list of all keys in the `keys` subsection.
++
+NOTE: The new version of Root metadata should be valid according to the rules of the previous and current versions. So you must sign Root metadata with a threshold of keys specified in the latest Root metadata file and a threshold of keys specified in the new Root metadata file. For more information, see the https://uptane.github.io/papers/uptane-standard.1.0.1.html#rfc.section.5.4.4.3[related section] of the Uptane standard.
+.. For the `threshold` object, specify the threshold number.
+
+** To change the threshold of Targets keys:
+.. In the `targets` subsection, for the `keyids` object, specify the list of valid keys that can be used for signing.
++
+You can find the list of all keys in the `keys` subsection.
+.. For the `threshold` object, specify the threshold number.
+
+. For the `version` object, specify the version number of the new Root metadata.
+. Sign the `root.json` file.
```

Comment:
Since we're talking about adding extra keys and using thresholds, the example should include adding at least one more signature.